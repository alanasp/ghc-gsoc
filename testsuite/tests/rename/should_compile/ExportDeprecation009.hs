{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

module Main where
import ExportDeprecation009_A
main = print b   -- b is deprecated
