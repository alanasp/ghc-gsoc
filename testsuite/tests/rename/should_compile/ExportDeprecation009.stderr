ExportDeprecation009.hs:5:14: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘b’
    (imported from ExportDeprecation009_A, but defined in ExportDeprecation009_B):
    Deprecated: "Import B directly"
